addons/im_livechat/static/src/core/web/livechat_channel_info_list.js

```diff
@@ -0,0 +1,22 @@
+import { ActionPanel } from "@mail/discuss/core/common/action_panel";
+import { Component } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { createDocumentFragmentFromContent } from "@mail/utils/common/html";
+
+export class LivechatChannelInfoList extends Component {
+    static components = { ActionPanel };
+    static template = "im_livechat.channelInfoList";
+    static props = ["thread"];
+
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+    }
+
+    onBlurNote() {
+        this.orm.write("discuss.channel", [this.props.thread.id], {
```

Comment:
Write is not allowed for most users (like demo user)

![image](https://github.com/user-attachments/assets/48cf3d84-5fde-471f-8ca8-f247695c43ef)

We should have a custom route that writes the field as sudo when the user can access (read) the channel.